### PR TITLE
Fr Translation

### PR DESCRIPTION
**Pull request by LapinFou**
Fixed few typos and add 1 missing translation.
Doesn't really need a review.

BTW, what does mean the abbreviation **CFN** in the _CustomFunctionData_?
It is in the file **companion\src\firmwares\customfunctiondata.cpp**, line n°248.
I can't translate it.

### CODE REVIEW

**Comment by kilrah** [comment]
CustomFunctionN... That was used in code usually, if it's a user-visible string it should be changed to SF

**Comment by kilrah** [comment]
It's actually used to describe both SF and GF, which is detailed later in the name.

clipboard01

**Comment by LapinFou** [comment]
Thanks Kilrah. 👍 

**SF** is translated into **FS**
**GF** is translated into **FG**
My suggestion would be "Custom Function N" translates in "Fonction Personnalisée N", so **CFN** → **FPN**

Any suggestion for the French translation?

**Comment by kilrah** [comment]
GF should probably be FG...

OK for FPN...

**Comment by LapinFou** [comment]
It is **FG**. I corrected my typo. 😃 
I will push the modification.

**Comment by LapinFou** [comment]
CFN "Custom Function N" has been translated in "Fonction Personnalisée N".
**CFN** → **FPN**

File: companion/src/translations/companion_fr.ts [hunk 1]
```diff
@@ -1503,12 +1503,12 @@ Voulez-vous les importer ?</translation>
...
         <location filename="../firmwares/customfunctiondata.cpp" line="94"/>
         <source>SetFailsafe Int. Module</source>
-        <translation>Déf. failsafe module interne</translation>
+        <translation>Définir failsafe module interne</translation>
     </message>
     <message>
         <location filename="../firmwares/customfunctiondata.cpp" line="96"/>
         <source>SetFailsafe Ext. Module</source>
-        <translation>Déf. failsafe module externe</translation>
+        <translation>Définir. failsafe module externe</translation>
```
Comment (kilrah) [inline companion/src/translations/companion_fr.ts]: Pourquoi le '.' ?

**Comment by LapinFou** [comment]
Nice catch. 😄

**Pull request merged**